mkdocs.yml

```diff
@@ -103,37 +102,40 @@ markdown_extensions:
 # Page tree
 nav:
   - Overview:
-    - Introduction: index.md
-    - Community: community.md
-    - Conceptual: conceptual.md
+      - Introduction: index.md
+      - Community: community.md
+      - Conceptual: conceptual.md
   - Getting Started:
-    - Deployment: deployment.md
-    - Kubernetes: ./deployment/kubernetes.md
-    - Docker Swarm: ./deployment/docker-swarm.md
-    - Docker Playground: ./deployment/play-with-docker.md
-    - Troubleshooting: ./deployment/troubleshooting.md
+      - Deployment: deployment.md
+      - Kubernetes: ./deployment/kubernetes.md
+      - Docker Swarm: ./deployment/docker-swarm.md
+      - Docker Playground: ./deployment/play-with-docker.md
+      - Troubleshooting: ./deployment/troubleshooting.md
   - CLI:
-    - Installation: ./cli/install.md
-    - Create functions: ./cli/templates.md
-    - Build functions: ./cli/build.md
-    - Working with tags: ./cli/tags.md
+      - Installation: ./cli/install.md
+      - Create functions: ./cli/templates.md
+      - Build functions: ./cli/build.md
+      - Working with tags: ./cli/tags.md
   - Tutorials:
-    - Workshop: ./tutorials/workshop.md
-    - CLI with Node.js: ./tutorials/CLI-with-node.md
-    - First Python Function: ./tutorials/first-python-function.md
-    - Test Drive: ./tutorials/test-drive.md
-    - Featured: ./tutorials/featured.md
+      - Workshop: ./tutorials/workshop.md
+      - CLI with Node.js: ./tutorials/CLI-with-node.md
+      - First Python Function: ./tutorials/first-python-function.md
+      - Test Drive: ./tutorials/test-drive.md
+      - Featured: ./tutorials/featured.md
   - OpenFaaS Cloud: openfaas-cloud/intro.md
   - Reference:
-    - Secrets: ./reference/secrets.md
-    - Auth: ./reference/authentication.md
-    - CORS: ./reference/cors.md
-    - Cron: ./reference/cron.md
-    - Triggers: ./reference/triggers.md
-    - YAML: ./reference/yaml.md
+      - Secrets: ./reference/secrets.md
+      - Auth: ./reference/authentication.md
+      - CORS: ./reference/cors.md
+      - Cron: ./reference/cron.md
+      - Triggers: ./reference/triggers.md
+      - YAML: ./reference/yaml.md
+      - Ingress:
```

Comment:
Ingress is probably not needed. I think SSL at the top level is OK. Also Ingress is Kubernetes-specific jargon that our users may not know or be familiar with. 

There will be another page here for Swarm so the sub-headings should be:

- Swarm
- Kubernetes